.gitignore

```diff
@@ -0,0 +1,160 @@
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```

Comment:
Your merge requested is still from master branch - push the code to the develop branch and change the source branch to develop